com.github.Inochi2D.Inochi-Creator.yml

```diff
@@ -0,0 +1,164 @@
+id: com.github.Inochi2D.Inochi-Creator
+runtime: org.kde.Platform
+runtime-version: "5.15-21.08"
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.ldc
+  - org.freedesktop.Sdk.Extension.dmd
+command: "inochi-creator"
+finish-args:
+    - "--device=all" #OpenGL rendering, webcams (may be required? not sure)
+    - "--socket=x11"
+    - "--share=ipc"
+    - "--socket=wayland"
+    - "--filesystem=host" #This is a dev tool, it should probably have as much access to the host as it can.
+    - "--filesystem=home" #So that it can access the project files, should they be located in the home folder
+    - "--filesystem=~/inochi-creator-crashdump.txt:rw" # Error logs
+#    - "--filesystem=~/.config/inochi-creator:rw" #configuration files, not needed, as Inochi follows the appropriate environment variables.
+    # - "--socket=session-bus"
+    - "--socket=system-bus"
+modules:
+  # --- Tools ---
+  - name: xprop
+    sources: 
+      - type: archive
+        url: https://xorg.freedesktop.org/archive/individual/app/xprop-1.2.4.tar.bz2
+        sha256: "8c77fb096e46c60032b7e2bde9492c3ffcc18734f50b395085a5f10bfd3cf753"
+
+  - name: zenity
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/zenity/3.41/zenity-3.41.0.tar.xz
+        sha256: 19b676c3510e22badfcc3204062d432ba537402f5e0ae26128c0d90c954037e1
+
+  - name: kdialog
+    buildsystem: cmake
+    sources:
+      - type: git
+        url: https://github.com/KDE/kdialog.git
+
+
+  # --- Inochi Creator ---
+  - name: inochi-creator
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/ldc/bin
+    build-commands:
+      - dub add-local ./deps/bindbc-imgui/ 0.7.0+build.20.g237a209;
+        dub add-local ./deps/facetrack-d/ 0.6.2+build.32.g9e1b2b1;
+        dub add-local ./deps/fghj/ 1.0.0;
+        dub add-local ./deps/gitver/ 1.5.0;
+        dub add-local ./deps/i18n/ 1.0.1+build.1.ge4a7f0b;
+        dub add-local ./deps/inmath/ 1.0.3+build.4.gec62993;
+        dub add-local ./deps/inochi2d/ 0.7.2+build.17.g0e32b36;
+        dub add-local ./deps/psd-d/ 0.6.2;
+        dub add-local ./deps/vmc-d/ 1.1.1+build.4.gb32fb96;
+        dub add-local ./deps/bindbc-loader/ 1.0.1;
+        dub add-local ./deps/bindbc-opengl/ 1.0.2;
+        dub add-local ./deps/bindbc-sdl/ 1.1.3;
+        dub add-local ./deps/imagefmt/ 2.1.2;
+        dub add-local ./deps/mir-algorithm/ 3.14.11;
+        dub add-local ./deps/mir-core/ 1.1.109;
+        dub add-local ./deps/semver/ 0.3.4;
+        dub add-local ./deps/silly/ 1.1.1;
+        dub add-local ./deps/tinyfiledialogs/ 0.10.1;
+        dub build --skip-registry=all --compiler=ldc2 --build=release --config=full;
+      - install -m 755 ./out/inochi-creator /app/bin/inochi-creator

```

Comment:
so I guess you don't install mandatory icons, .desktop, appdata files.

Reply:
Indeed, I have not yet sorted those out. I want to get the app in a functioning state without them first.

Reply:
it's required for a build